HopsanGUI/LibraryHandler.cpp

```diff
@@ -82,6 +82,15 @@
 
 namespace  {
 
+QString visibleFilePath(const QFileInfo & fi) {
```

Comment:
Perhaps add a description comment to this function. I didn't understand the purpose of it until I read the commit message.

Reply:
Yes, will do